src/storage/settings/settingsPersistentStorage.ts

```diff
@@ -49,7 +49,8 @@ export const defaultSettings: Settings = {
     discreetMode: false,
     passwordRequirement: false,
     language: undefined,
-    devTools: false
+    devTools: false,
+    analytics: true
```

Comment:
@mvaivre I wonder what your thoughts on this are. Do we enable analytics by default?

I did a fresh install of Ledger Live on my computer. It was enabled by default and there was no consent asked.

Based on GDPR, we [MUST acquire unambiguous consent](https://posthog.com/docs/privacy/gdpr-compliance#you-must-acquire-unambiguous-consent). I am not sure whether that only applies when we capture [personal data (or PII - Personal Identifiable Information)](https://posthog.com/blog/what-is-personal-data-pii), or if it applies in all cases. [This suggests](https://posthog.com/docs/privacy/gdpr-compliance#step-4-configure-consent) that only if we collect personal data do we need to ask for consent, but it's not 100% clear. To the best of my research and knowledge, I have configured out analytics to not capture personal data.

> If you require robust GDPR compliance, we recommend using [PostHog Cloud EU](https://posthog.com/eu) – a managed version of PostHog that's hosted on servers based in Frankfurt.

✅ we do that already.

Reply:
@mvaivre Wait, I didn't get your feedback here 😅

Reply:
We don't collect any personal data & ledger activates this by default => I think we're good to go.

Now, if we want to make extra sure we're fine, we could add a consent box during the onboarding (when creating or importing a wallet for the first time), at the very last step with the confettis. The "let's go" button could be deactivated until the user accept the condition. Should we open an issue and potentially plan this for a following release?

Reply:
> The "let's go" button could be deactivated until the user accept the condition.

You mean the user wouldn't be able to use the wallet without enabling analytics? I guess this is not what you mean cause that sounds bad.

> Should we open an issue and potentially plan this for a following release?

Let's do that.